From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jgold6 committed:
The Getting Started Download has been moved

It is no longer in the Mobile Section. It i snow in the Deployment section of the settings blade. Also it no longer says "Getting Started" it now says "Quickstart".

jolleymd committed:
Update AppGW SSL Policy notes and process

Need to make sure users know that the AppGW SSL Policy cmdlets have parameters that cannot mix based on the PolicyType specified. Also added section for updating an AppGW since users tend to miss the Set for the AppGw, itself.

georgewallace committed:
Made additional updates to the PR.

tynevi committed:
Merge branch 'patch-8' of https://github.com/georgewallace/azure-docs into jolly

tynevi committed:
Update docfx.json

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-711

pull upstream master

tynevi committed:
resolving mc from public pr #2648

Commit:
task 711: add SAS storage support article

Commit:
Accelerated networking-Linux

Commit:
Accelerated networking-Linux

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

Commit:
task 711: update screen shots and add outlines

mumian committed:
new Power BI connection article

mumian committed:
update

Commit:
task 711: update screen shots and add example

Commit:
task 711: update screen shot and examples

Commit:
Updates.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

Commit:
task 711: fix code formatting

Commit:
task 711: edits

Commit:
task 711: edits

mumian committed:
update an image

Commit:
Added Windows article.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

Commit:
Fixes.

Commit:
Fixes.

Commit:
task 711: incorporate review comments

Commit:
task 711: Incorporate more review comments, add to TOC

Commit:
Updated article and code.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into azTransfer

Commit:
Updating with Sercan's feedback.

Commit:
Re-working sentence

Commit:
Additional Sercan feedback.

Commit:
Fixing sentence.

Commit:
Fixing paragraphs.

Commit:
task 711: Add URL Rewrite rule examples

Commit:
task 711: Update URL Rewrite rule examples

Commit:
Adding Wei Wei's feedback.

Commit:
task 711: Edits

Commit:
Adding failure input

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into azTransfer

Commit:
Acrolinx feedback.

Commit:
Tags

Commit:
Some retructuring.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into azTransfer

Commit:
Improvements.

Commit:
Rewrites.

Commit:
Updated version checker.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into azTransfer

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

Commit:
PM fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

Commit:
task 711: incorporate review comments - fix sample SAS url

cjgronlund committed:
freshness review complete

cjgronlund committed:
fresh review complete

cjgronlund committed:
spelling correx

cjgronlund committed:
freshness updates

cjgronlund committed:
updates

Commit:
task 711: replace with higher-res screenshot

Commit:
task 711: fix outline

Commit:
task 711: remove duplicate info

Commit:
task 711: fix TOC node

mumian committed:
add cross-links

mumian committed:
edit toc

Commit:
task 711: Change title for SEO, update headings

Commit:
Changes

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

raiye committed:
Update cloud-services-guestos-update-matrix.md

neilpeterson committed:
ACI link update

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

Commit:
KRIS: MVC metadata update

billmath committed:
removing note

Commit:
Fixes.

Commit:
Updates.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

Lisaco88 committed:
Merge pull request #30267 from dksimpson/dean-711

Dean 711

mumian committed:
refresh

Commit:
clf edit

Commit:
Fixing script grammar.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into azTransfer

Lisaco88 committed:
Merge pull request #30747 from raiye/patch-1

Update cloud-services-guestos-update-matrix.md

Lisaco88 committed:
Merge pull request #30749 from neilpeterson/aci-link-fix

ACI link update

mtillman committed:
Add link to RSS icon

tynevi committed:
Update app-service-mobile-xamarin-forms-get-started.md

HJToland3 committed:
article title changes

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

GitHubber17 committed:
corrected year in ms.date metadata

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

Lisaco88 committed:
Merge pull request #30617 from roygara/azTransfer

Updating article

GitHubber17 committed:
Merge pull request #30754 from mumian/0103-freshness

refresh

tynevi committed:
Merge pull request #29992 from tysonn/2648

2648

tynevi committed:
Merge pull request #25644 from tysonn/jolly

Bringing in changes from public repo

GitHubber17 committed:
Merge pull request #30755 from raymondlaghaeian/patch-21

clf edit

GitHubber17 committed:
Merge pull request #30752 from billmath/pta

removing note

yujhongmicrosoft committed:
Update documentation-government-developer-guide.md

GitHubber17 committed:
Merge pull request #30756 from mtillman/patch-3

Add link to RSS icon

Commit:
Freshness

mumian committed:
resolve conflic

Commit:
acrolinx

yujhongmicrosoft committed:
Update documentation-government-developer-guide.md

Commit:
table

Lisaco88 committed:
Merge pull request #30751 from kriscrider/mvc-overview-1.3

KRIS: MVC metadata update

sanjeev3 committed:
Update api-reference.md

GitHubber17 committed:
Merge pull request #30757 from HJToland3/master

Online migration article title updates

swkrish committed:
Merge pull request #18 from MicrosoftDocs/master

update private

GitHubber17 committed:
Merge pull request #30758 from yujhongmicrosoft/patch-10

Update documentation-government-developer-guide.md

Commit:
A few edits.

sanjeev3 committed:
Update api-reference.md

Commit:
PR reviewer edits.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

swkrish committed:
Update active-directory-aadconnect-sso-faq.md

swkrish committed:
Update active-directory-aadconnect-sso-faq.md

Commit:
date

Commit:
PR reviewer edits.

ktoliver committed:
Merge pull request #30768 from jimdial/patch-52

A few edits.

Commit:
Edit.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into accelerated-dpdk

tynevi committed:
Merge pull request #30273 from jimdial/accelerated-dpdk

Accelerated dpdk

Ja-Dunn committed:
Merge pull request #30378 from mumian/1219-iq-powerbi

new Power BI connection article

tynevi committed:
Merge pull request #25716 from MicrosoftDocs/docfxupdate

Update docfx.json

ktoliver committed:
Merge pull request #30764 from sanjeev3/master

Update api-reference.md

ktoliver committed:
Merge pull request #30770 from swkrish/master

Seamless SSO - cloud env support

Ja-Dunn committed:
Merge pull request #30760 from cynthn/migrate-to-managed-disks

Freshness - migrate to managed disks

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Ja-Dunn committed:
Merge pull request #30719 from cjgronlund/fresh-jan

freshness review complete

v-alje committed:
Merge pull request #30779 from MicrosoftDocs/FromPublicRepo

From public repo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
